src/js/store/orgs/actions.ts

```diff
@@ -110,3 +132,127 @@ export const provisionFailed = ({
     payload: model,
   });
 };
+
+export const refetchOrg = (org: Org): ThunkResult => async dispatch => {
+  const url = window.api_urls.scratch_org_detail(org.id);
+  dispatch({
+    type: 'REFETCH_ORG_STARTED',
+    payload: { org, url },
+  });
+  try {
+    /* istanbul ignore if */
+    if (!url) {
+      throw new Error(`No URL found for org: ${org.id}`);
+    }
+    const response = await apiFetch({ url, dispatch });
+    if (!response) {
+      return dispatch({
+        type: 'REFETCH_ORG_FAILED',
+        payload: { org, url, response },
+      });
+    }
+    // @@@ Mock out until API exists
+    // setTimeout(() => {
+    //   dispatch({
+    //     type: 'SCRATCH_ORG_UPDATED',
+    //     payload: { ...response },
+    //   });
+    // }, 3000);
```

Comment:
Can we clear out this comment?

Reply:
Nope, not until you make the `GET` endpoint actually check for updated changes. :)

I expect it'll get cleared in the next PR -- for now it's helpful in dev.